Corona-Warn-App/src/main/assets/privacy_de.html

```diff
@@ -759,12 +795,9 @@ <h1>
     </li>
     <li>
         das Recht, den behördlichen
-		<a
-					href="https://www.rki.de/DE/Content/Institut/OrgEinheiten/Datenschutz/Datenschutz_node.html"
-				>
         Datenschutzbeauftragten des RKI
-		</a>
```

Comment:
Why was the link deleted? If I recall correctly, this was changed in the last update.

Reply:
I wasn't aware of this discussion, but also if it's included the link is still not clickable even if it looks like that.

So I would add the link again in the HTML, but we should still should check why it isn't clickable.

https://user-images.githubusercontent.com/64483219/105026600-018f0b00-5a4f-11eb-95c8-2f431a6aa6b2.mp4